# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/dietlibc/pkg_patch/pkg_cpio.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- cpio-2.5/rtapelib.c~	Thu Dec  6 05:55:12 2001
+++ cpio-2.5/rtapelib.c	Thu Aug 21 01:17:19 2003
@@ -202,7 +202,8 @@
   return atoi (cp + 1);
 }
 
-#ifdef HAVE_NETDB_H
+#if 0
+/*#ifdef HAVE_NETDB_H*/
 /* Execute /etc/rmt as user USER on remote system HOST using rexec.
    Return a file descriptor of a bidirectional socket for stdin and stdout.
    If USER is NULL, or an empty string, use the current username.
@@ -340,7 +341,8 @@
     }
   *dev = '\0';
 
-#ifdef HAVE_NETDB_H
+#if 0
+/*#ifdef HAVE_NETDB_H*/
   /* Execute the remote command using rexec.  */
   READ (i) = WRITE (i) = _rmt_rexec (system, login);
   if (READ (i) < 0)
